src/ameba/rule/style/redundant_self.cr

```diff
@@ -0,0 +1,110 @@
+require "compiler/crystal/syntax/token"
```

Comment:
why do we need to have this here?

Reply:
Because of `CRYSTAL_KEYWORDS = Crystal::Keyword.values.map(&.to_s)` declaration